### PR TITLE
[DEV APPROVED] change to land transaction calculator text

### PR DESCRIPTION
**Pull request by akenzua**
This PR changes the text of the land transaction calculator in accordance with the requirements of [TP11346](https://maps.tpondemand.com/restui/board.aspx?#page=userstory/11346)

### CODE REVIEW

**Comment by masbuild** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">There are code changes, but no corresponding tests. Please include tests if this PR introduces any modifications in behavior. If this is a refactor PR, please ignore the message.Refer to our mas standards guides about opening a PR<a href="link-to-the-mas-standards-about-opening-a-pr">here</a></td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">It looks like this is a Rails engine, but no changes to <a href="https://github.com/moneyadviceservice/mortgage_calculator/blob/b609ff83d484f80b44cc4c8749d0793463485c39/lib/mortgage_calculator/version.rb">lib/mortgage_calculator/version.rb</a> detected. Did you forget to bump the version?</td>
    </tr>
  </tbody>
</table>

At Money Advice Service, we use Danger gem, to ensure that our commits and pull requests follow [standards](https://github.com/moneyadviceservice/mas-standards/blob/master/Dangerfile). If you think one of the points here has been raised in error (or shouldn't be in our standards), open an issue or pull request on [our Danger config](https://github.com/moneyadviceservice/mas-standards).

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: config/locales/land_transaction_tax.en.yml [hunk 1]
```diff
@@ -8,26 +8,26 @@ en:
...
     title: "Calculate the Land Transaction Tax on your residential property in Wales"
     next: Next
     select:
       label: I am
       option_prompt: please select an option
       option_isNextHome: buying my next home
-      option_isSecondHome: buying an additional or buy-to-let property
-      tooltip_html: "Anyone purchasing an additional home including buy to let properties over the value of £40,000 will have to pay an extra 3% surcharge on top of each Land Transaction Tax band."
+      option_isSecondHome: buying an additional property or second home
+      tooltip_html: "You will usually have to pay higher rates of Land Transaction Tax if you are buying a residential property worth %{amount} or more and already own 1 or more properties."
```
Comment (davidtrussler) [inline config/locales/land_transaction_tax.en.yml]: We are not passing a value here for `%{amount}` so the view is rendering this literally. We should either ensure a value is passed form the view or we use the figure in the YAML.  
We also need to make this consistent across the English and Welsh language versions, which they are not as it stands (the Welsh language YAML contains the figure).

File: config/locales/land_transaction_tax.cy.yml [hunk 2]
```diff
@@ -42,12 +44,14 @@ cy:
...
       rate_header: "Cyfradd Treth Trafodion Tir ac Adeiladau (LBTT)"
-      extra_rate_header: "Atodiad Prynu-i-osod / Annedd Atodol (ADS)*"
+      extra_rate_header: "Cyfradd LTT Uwch*"
       max: "%{value}+"
     next_steps:
       learn_more:
         title: "A wyddech chi?"
-        tip_1: "Rhaid i chi dalu Treth Trafodiadau Tir cyn pen 14 diwrnod ar ôl prynu eiddo. Os ydych yn defnyddio cyfreithiwr i gwblhau’r trawsgludo, bydd fel arfer yn trefnu’r taliad ar eich rhan."
+        tip_1: "Mae’n rhaid i chi anfon ffurflen LTT a thalu'r dreth cyn pen 30 diwrnod o'r diwrnod ar ôl i chi gwblhau (neu ddyddiad effeithiol arall y trafodiad). Os oes gennych gyfreithiwr, asiant neu drawsgludwr, byddant fel arfer yn ffeilio'ch ffurflen ac yn talu'r dreth ar eich rhan. Os na wnânt hyn ar eich rhan, gallwch ffeilio ffurflen a thalu'r dreth eich hun gan ddefnyddio'r"
+        tip_1_link: "https://gov.wales/paper-return-land-transaction-tax"
```
Comment (davidtrussler) [inline config/locales/land_transaction_tax.cy.yml]: This should use a Welsh language URL.

File: app/views/mortgage_calculator/land_transaction_taxes/_results_recalculation_form.html.erb [hunk 3]
```diff
@@ -87,6 +87,7 @@
     </h2>
     <p class="stamp-duty__info-tip">
       <%= I18n.t("land_transaction_tax.next_steps.learn_more.tip_1") %>
+      <%= link_to  I18n.t("land_transaction_tax.next_steps.learn_more.tip_1_link_url") , I18n.t("land_transaction_tax.next_steps.learn_more.tip_1_link"), :target => "_blank" %>
```
Comment (davidtrussler) [inline app/views/mortgage_calculator/land_transaction_taxes/_results_recalculation_form.html.erb]: It's not critical, but our usual practice here has been to write this as something like: `<%= t("land_transaction_tax.next_steps.learn_more.tip_1_html", href: t('land_transaction_tax.next_steps.learn_more.tip_1_link')).html_safe %>` i.e. passing the URL from the view to the YAML. Then the translation file would need to include the markup, something like: `using the <a href="%{href}" target="_blank">paper return</a>.`

**Comment by akenzua** [comment]
Sorted.

**Comment by davidtrussler** [comment]
There are some cucumber tests failing as a result of these changes. Run `bundle exec cucumber` to find the problem: most likely some values have changed and need updating in a test.

**Comment by davidtrussler** [comment]
This are what appear to me to be problems with the copy changes in the requirements for this card that I am querying in the TP ticket. This is nothing to do with this PR but may mean it's worth waiting for a response on this before proceeding.

File: features/step_definitions/stamp_duty.rb [hunk 4]
```diff
@@ -62,10 +62,6 @@
   @stamp_duty.select('buying my next home', from: @buyer_type)
 end
 
-Given("I am an additional or buy-to-let property buyer") do
-  @stamp_duty.select('buying an additional or buy-to-let property', from: @buyer_type)
-end
```
Comment (davidtrussler) [inline features/step_definitions/stamp_duty.rb]: Ha, my bad, looks like this is still needed for the Scottish version of the tool: the test fails elsewhere without it. If you restore this I think all the tests run fine.

File: config/locales/land_transaction_tax.en.yml (hunk 1 above)
Comment (davidtrussler) [reply config/locales/land_transaction_tax.en.yml]: This is still rendering as `%{amount}`, but in both languages now. It needs to be passed through from the view, like `text: t("land_transaction_tax.select.tooltip_html", amount: '£40,000')` in _calculator_form.html.erb

**Comment by kiggundu** [comment]
We need to up the version of the gem before merging

**Pull request merged**